pkg/config/config.go

```diff
@@ -170,6 +171,8 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("health_port", int64(0))
 	config.BindEnvAndSetDefault("disable_py3_validation", false)
 	config.BindEnvAndSetDefault("python_version", DefaultPython)
+	config.BindEnvAndSetDefault("iot_host", AgentFlavor == "datadog-iot-agent")
```

Comment:
I would create constants somewhere, the same way there is this: https://github.com/DataDog/datadog-agent/blob/master/pkg/aggregator/aggregator.go#L47-L56
These values may even be re-usable but they would be misplaced then.